src/asInput/index.jsx

```diff
@@ -22,6 +24,7 @@ export const inputProps = {
   onBlur: PropTypes.func,
   validator: PropTypes.func,
   className: PropTypes.arrayOf(PropTypes.string),
+  theme: PropTypes.arrayOf(PropTypes.string),
```

Comment:
Naming nit: if it is an array of `string`s should this be called `themes` instead?

Reply:
Good catch!